docs/documentation.md

```diff
@@ -570,9 +570,9 @@ You can follow along [on my technical blog](https://ttboj.wordpress.com/).
 
 To report any bugs, please file a ticket at: [https://github.com/purpleidea/mgmt/issues](https://github.com/purpleidea/mgmt/issues).
 
-##Authors
+## Authors
 
-Copyright (C) 2013-2016+ James Shubin and the project contributors
+Copyright © 2013-2016+ James Shubin and the project contributors
```

Comment:
@purpleidea,
https://guides.github.com/features/mastering-markdown/
http://spec.commonmark.org/0.27/
https://daringfireball.net/projects/markdown/basics — original markdown.pl
Headers with spaces works in every markdown implementations, it's the right syntax.